templates/4_crystallicum_dark_templates/usercp/usercp_options.html

```diff
@@ -12,24 +12,15 @@
 <div id="content_width">
 <td valign="top">
 {$errors}
-<table border="0" cellspacing="{$theme['borderwidth']}" cellpadding="{$theme['tablespace']}" class="tborder">
+<table border="0" cellspacing="{$theme['borderwidth']}" cellpadding="{$theme['tablespace']}" class="tborder ucp_avtable ucp_input">
 <tr>
 <td class="thead" colspan="2"><strong>{$lang->edit_options}</strong></td>
 </tr>
 <tr>
 <td width="100%" class="trow1" valign="top">
-<fieldset class="trow2">
-<legend><strong>{$lang->login_cookies_privacy}</strong></legend>
```

Comment:
This was the field "Nie pokazuj mnie na liście osób online". We are removing it because setting it in ACP on mybb doesn't work, and even if you select the option to not allow user to choose that option, they still can. So the way to make sure the option which we think for our purposes is not useful is to just remove it from the UCP.